theme/ckeditor5-media-embed/mediaform.css

```diff
@@ -29,7 +29,7 @@
 		padding: 0;
 		width: calc(.8 * var(--ck-input-text-width));
 
-		& .ck-labeled-input {
+		& .ck-labeled-field-view {
```

Comment:
Something's wrong with the media form

![image](https://user-images.githubusercontent.com/1099479/78666654-f1e77100-78d7-11ea-9a31-bc735ec2d40d.png)

and when I remove this rule it goes back to normal:

![image](https://user-images.githubusercontent.com/1099479/78666692-0297e700-78d8-11ea-8699-4f01a70c55bc.png)

but I see that all forms (link, text alternative) have this style and it works.

--- 

Anyway, it is caused by the status text which does not wrap to the next line and removing it "fixes" the problem

![image](https://user-images.githubusercontent.com/1099479/78666988-7803b780-78d8-11ea-9956-2fc09aab88d1.png)

![image](https://user-images.githubusercontent.com/1099479/78667007-818d1f80-78d8-11ea-8f70-c4bbbd63c8a0.png)

Why it worked before?

Reply:
We also lost all the colors and font size in the status:

**Expected:**
![image](https://user-images.githubusercontent.com/1099479/78667245-e34d8980-78d8-11ea-899f-83720f94cf85.png)

**Actual:**

![image](https://user-images.githubusercontent.com/1099479/78667336-0c6e1a00-78d9-11ea-9358-fb15114b564d.png)

---

It could be that because the font was smaller, issues I mentioned in my previous comment were never visible.

Reply:
Same in the table props dialog, font size is wrong

![image](https://user-images.githubusercontent.com/1099479/78667521-64a51c00-78d9-11ea-9403-3d4a8e2f3485.png)